python/lsst/ap/association/afwUtils.py

```diff
@@ -805,7 +805,7 @@ def get_ccd_visit_info_from_exposure(exposure):
     #  flux zero point error ``counts``]
     values = {'ccdVisitId': visit_info.getExposureId(),
               'ccdNum': exposure.getDetector().getId(),
-              'filterName': filter_obj.getName(),
+              'filterName': filter_obj.getCanonicalName(),  # use abstract filter
```

Comment:
`get_ccd_visit_info_from_exposure` doesn't seem to be used anywhere; can it be removed?

Reply:
Yes it can. It's left over from a previous iteration of the Apdb and, as you point out, is not used anywhere.